app/web/src/components/Navigation/Navigation.tsx

```diff
@@ -163,6 +254,69 @@ export default function Navigation() {
     </div>
   );
 
+  const menuItems = menuDropDown(data).map(
+    ({
+      name,
+      notificationCount,
+      route,
+      target,
+      externalLink,
+      hasBottomDivider,
+    }) => {
+      const hasNotification =
+        notificationCount !== undefined && notificationCount > 0;
+
+      const linkContent = (
+        <>
+          {hasNotification ? (
+            <Badge color="primary" variant="dot" className={classes.badge}>
+              <Typography noWrap>{name}</Typography>
+            </Badge>
+          ) : (
+            <Typography noWrap>{name}</Typography>
+          )}
+
+          {hasNotification ? (
+            <Typography
+              noWrap
+              variant="subtitle2"
+              className={classes.notificationCount}
+            >
+              {`${notificationCount} unseen`}
+            </Typography>
+          ) : null}
+        </>
+      );
+
+      return (
+        <MenuItem
+          hasNotification={hasNotification}
+          hasBottomDivider={hasBottomDivider}
+        >
+          {externalLink ? (
+            <a
+              href={route}
+              key={name}
+              target={target}
+              onClick={() => setMenuOpen(false)}
```

Comment:
Okay REALLY last thing I promise, can you make this and the `Link` below have 100% width with a `className`. Right now they only work if you click the text, not the space on the right (and not the padding, but I think that's no big deal).

Reply:
Of course :smile:

Reply:
@lucaslcode in order to do that i need to remove padding on li which is on line 13 inside components/Menu.tsx, should i add another props and add a styles inside classNames of that component ? Or should i remove that paddingInline ? ( the latter may impact other part of the app. Because <a> and <Link> are actually 100% of their parent :monocle_face:

Reply:
I think what Lucas said should "just work" based on playing around the styles interactively in the browser. The `<a>`/`<Link>`s aren't 100% width since they don't have a width explicitly set at the moment

Reply:
I don't think they are?
![image](https://user-images.githubusercontent.com/32044095/146476767-f14364fd-9000-4e22-b67f-bd121a112e22.png)

Reply:
@lucaslcode my bad, i added the 100% locally and i meant about that green padding ^^ i'm pushing the changes